server/src/test/java/io/spine/server/storage/RecordStorageShould.java

```diff
@@ -656,7 +656,7 @@ public void read_both_by_columns_and_IDs() {
         assertEquals(1, readRecords.size());
         final EntityRecord readRecord = readRecords.get(0);
         assertEquals(targetEntity.getState(), unpack(readRecord.getState()));
-        assertEquals(targetId, unpack(readRecord.getEntityId()));
+        assertEquals(targetId, Identifier.unpack(readRecord.getEntityId()));
```

Comment:
If possible, please import `Identifier.unpack` statically. If not, that's OK too.

Reply:
I cannot, it will lead to the ambiguous call between this method and `AnyPacker#unpack(Any)` which has wider usage in this class.